release-notes/major11/11.0.0/valtimo-backend-libraries.md

```diff
@@ -95,6 +95,20 @@ The following breaking changes were introduced:
   
   `valtimo.annotation-scan.accepted-packages: test.package1, test.package2`
 
+* **Deleted 'Authority' entities**
+  Valtimo has switched to a new policy based access control system. As a consequence, the old authorities have been
+  removed. The following entities can no longer be used:
+  - Database table: 'jhi_authority'
+  - Spring events: AuthorityCreatedEvent, AuthorityDeletedEvent, AuthorityEvent, AuthorityNameChangedEvent
+  - Spring beans: AuthorityResource, AuthorityService, AuthorityRepository
+  - Domain: Authority, AuthorityRequest, Money
+
+* **Deleted 'Context' entities**
```

Comment:
Do we need something about this in the migration notes as well?

Reply:
Can we have migration notes for this? Because there is no alternative

Reply:
Something like "If you are using these entities/services, please don't use them anymore" I think still counts as a migration note.